cmd/fluxctl/list_workloads_cmd.go

```diff
@@ -101,3 +107,18 @@ func policies(s v6.ControllerStatus) string {
 	sort.Strings(ps)
 	return strings.Join(ps, ",")
 }
+
+// Extract workloads having its container name equal to containerName
+func filterByContainerName(workloads []v6.ControllerStatus, containerName string) (filteredWorkloads []v6.ControllerStatus) {
+    for _, workload := range workloads {
+        if len(workload.Containers) > 0 {
+			for _, c := range workload.Containers {
```

Comment:
Not sure if looping on the list is the best way to test all the containers in a workload.

Reply:
Why not? I think that would be fine

Reply:
Well I was wondering if there was a built-in function to do that, as I am not a Go expert :sweat_smile: 

But it is working ok!